hooks/post_gen_project.py

```
@@ -37,23 +37,39 @@ def init_git():
         git.wait()
 
 def init_proto():
-    print ("Fetching go modules, might take a few minutes")
-    code = Popen(["go","mod", "download", "all"], cwd=PROJECT_DIRECTORY).wait()
+    print("Starting proto initialization...")
+    print("Step 1/5: Fetching Go modules (this might take a few minutes)...")
+    code = Popen(["go", "mod", "download", "all"], cwd=PROJECT_DIRECTORY).wait()
```

Ruff (0.14.14)

Starting a process with a partial executable path

(S607)